Source: Gentoo
Upstream: no
Reason: output format changed from gcc 4.2
diff -urN perl-5.8.8_orig/makedepend.SH perl-5.8.8/makedepend.SH
--- perl-5.8.8_orig/makedepend.SH	2003-06-05 20:11:10.000000000 +0200
+++ perl-5.8.8/makedepend.SH	2007-07-08 02:04:34.000000000 +0200
@@ -167,6 +167,7 @@
             -e '/^#.*<builtin>/d' \
             -e '/^#.*<built-in>/d' \
             -e '/^#.*<command line>/d' \
+	    -e '/^#.*<command-line>/d' \
 	    -e '/^#.*"-"/d' \
 	    -e '/: file path prefix .* never used$/d' \
 	    -e 's#\.[0-9][0-9]*\.c#'"$file.c#" \
